MODULE.bazel

```diff
@@ -0,0 +1,21 @@
+module(
+    name = "rules_rust",
+    version = "0.9.0",
+)
+
+bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "rules_cc", version = "0.0.1")
+bazel_dep(name = "bazel_skylib", version = "1.2.0")
+bazel_dep(name = "apple_support", version = "0.13.0")
+
+# Load tinyjson without using the process wrapper
+raw_cargo = use_extension("//rust/private:extensions.bzl", "raw_cargo")
+raw_cargo.crate(
+    name = "tinyjson",
+    sha256 = "1a8304da9f9370f6a6f9020b7903b044aa9ce3470f300a1fba5bc77c78145a16",
```

Comment:
These values should probably be shared with the non-bzlmod implementation

Reply:
PTAL at the PR I sent you. Amongst other things, it shares the values.

https://github.com/cameron-martin/rules_rust/pull/1/commits/0760cf7516a75eae59dbcf7f20316b6bb55642c6